### PR TITLE
Preview

### PR DESCRIPTION
**Pull request by JoshThomasField**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  <script src="https://kit.fontawesome.com/8eb762019d.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="color-pallet.css">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <nav>
+            <div class="mobile-nav">
+                <img class="logo" src="signature (1).png" alt="Logo">
```
Comment (jennymegan) [inline index.html:16]: I very much like this logo

File: index.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                    <a>Home</a>
+                    <a>Home</a>
+                    <a>Home</a>
+                </div>
+            </div>
+        </nav>
+        <section>
+            <div class="container display-container">
+                <span class="tags top-tags"> &nbsp;&nbsp;&nbsp;&lt;body&gt;</span>
+                <h1 aria-label=" Hi, I’m Jack,web developer." class="blast-root">
```
Comment (jennymegan) [inline index.html:38]: Why does this say Jack?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        <section>
+            <div class="container contact-form">
+                <form action="">
+                    <div class="input-group">
+                        <label for="fname">Name</label>
+                        <input type="text" id="fname" name="firstname" placeholder="Name">
+                    </div>
+                    <div class="input-group">
+                        <label for="fname">Email</label>
+                        <input type="text" id="fname" name="firstname" placeholder="Email">
```
Comment (jennymegan) [inline index.html:117]: For screen readers use better names in the labels?

Comment (jennymegan) [reply index.html:117]: Also when you come to connect up this form to PHP you'll need to name the inputs accurately, they all say "firstname" at the moment.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+
+/* Custom components */
+
+.mobile-nav {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
```
Comment (jennymegan) [inline style.css:61]: Spacing and indentation here could be tidied up (unless this is something github is doing in this display)

**Review by jennymegan — CHANGES_REQUESTED** [review]
Are the grey elements <h1> <body>"on the header for visiual effect?
If so there is a typo in <h1/>   - and maybe consider spacing them out and adding a few more, at the oemnt they look a bit like a error.

File: index.html (hunk 2 above)
Comment (JoshThomasField) [reply index.html:38]: Cause I'm a thief

File: style.css (hunk 4 above)
Comment (JoshThomasField) [reply style.css:61]: Just something the pre-fixer does, I think it's to clearly identify groups by properties.

**Comment by JoshThomasField** [comment]
> Are the grey elements ie h1 bodyon the header for visiual effect?
> If so there is a typo in h1/ - and maybe consider spacing them out and adding a few more, at the moment they look a bit like a error.

Definite a visual effect that I'm going to play with late.

**Review by jennymegan — APPROVED** [review]

**Pull request merged**